package.json

```diff
@@ -31,4 +31,4 @@
     "passport-local-mongoose": "^8.0.0",
     "sanitize-html": "^2.13.0"
   }
-}
+}
```

Comment:
Please don't make any changes in package.json

Reply:
yes yes